examples/templates/jfrog-docker/main.tf

```
@@ -10,13 +10,23 @@ terraform {
     }
     artifactory = {
       source  = "registry.terraform.io/jfrog/artifactory"
-      version = "6.22.3"
+      version = "~> 8.4.0"
     }
   }
 }
 
 locals {
-  username = data.coder_workspace.me.owner
+  # if the jfrog username is same as the coder username, you can use the following
+  # artifactory_username = data.coder_workspace.me.owner
+  # if the username is same as email, you can use the following
+  # artifactory_username = urlencode(data.coder_workspace.me.owner_email)
+  artifactory_username = data.coder_workspace.me.owner
+  artifactory_registry_keys = {
+    "npm"    = "npm"
+    "python" = "python"
```

In the example in the docs you used `pypi`, which is also the default name in Artifactory, and I think it is better to keep the two examples (docs and this full example) aligned.
```suggestion
    "pypi" = "pypi"
```

---

Thanks for the useful commnets. `pypi` is the key for a remote repository while `python` is a virtual repository that inlcludes `pypi`. I will sunchronise docs and template to use same keys.